### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'lng')

### ISSUE THREAD

**Issue by pawelpepek** [flsy/react-mapycz#405]
Hi,
I'm new in the React.

I had to install an older React version
`"react": "^16.13.1"` 
because I had a conflict with this library (is this required?).

After this everything seemed to be ok. In the browser "Loading..." appeared, but after this I had errors:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'lng')
    at Map.tsx:46
```

My code is simply:
```
import React from "react"
import "./App.css"
import { Map } from 'react-mapycz'

function App() {
	return (
		<Map />
	)
}

export default App
```

Best regards,

**Comment by CrossTheDev** [comment]
`Center` prop is actually required. Documentation should be corrected or default `center` implemented.

**Comment by kolebjak** [comment]
```

"peerDependencies": {
    "react": "^16.13.1",
    "react-dom": "^16.13.1"
  }

**Comment by kolebjak** [comment]
I've created PR for optional `center` Map property. This error shouldn't occur after it's merged and released. I've also extended landing page with short info about `<Map />` element (not complete, but better than nothing ). 

https://github.com/flsy/react-mapycz/pull/406

**Comment by kolebjak** [comment]
Should be fixed in `1.1.20`. Feel free to reopen this issue if there is still a problem or create a new one if you find different one. Cheers.
